### DISCUSSION TITLE
Local Client Lock

### DISCUSSION THREAD

**Discussion by SheiylaDev** [AdguardTeam/AdGuardHome#5543] [Ideas]
Hello, I installed AdGuard Home on a PI in router mode, I am not disappointed because it works very well. 

However there is a point that will be good to add, it is the possibility of accepting or not that the client that comes from my local network range because I find myself with external ip that connects without my permission.

**Comment by ainar-g** [comment]
This sounds a lot like the already existing *Access settings* feature, which you can setup on the *DNS settings* page.

**Reply by SheiylaDev** [reply]
I added 192.168.0.1/24 in the authorized clients, is that right ?
